README doesn't document the need for `schemaVersion: "2.0.0"`

README doesn't document the need for `schemaVersion: "2.0.0"`

```
============================================================
====== Test file: container_structure_test_linux.yaml ======
============================================================
=== RUN: ‌
--- FAIL‌
Error: error parsing config file: Please provide JSON schema version‌


=============================================================
========================== RESULTS ==========================
=============================================================
Passes:      0‌
Failures:    1‌
Total tests: 1‌

FAIL‌
time="2019-02-20T01:29:16Z" level=fatal msg=FAIL
##[error]Bash exited with code '1'.
##[error]Bash wrote one or more lines to the standard error stream.
```

--- a/README.md
+++ b/README.md
@@ -66,7 +66,7 @@ as normal.
 
 #### Supported Fields:
 
-This is the current schema version (v2.0.0).
+**NOTE: `schemaVersion` must be specified in all container-structure-test yamls. The current version is `v2.0.0`.**
 
 - Name (`string`, **required**): The name of the test
 - Setup (`[][]string`, *optional*): A list of commands

